client/components/Sponsor/CardContainer/Card/Card.css

```diff
@@ -0,0 +1,41 @@
+.sponser-card{
+	border-radius: 4px;
+    border: 1px solid rgba(0,0,0,.15);
+    margin: 14px;
+}
+
+.sponser-card-content{
+	padding: 12px;
+}
+
+.sponser-card-content p{
+	margin: 0;
+}
+
+#sponser-card-content-name{
+	font-size: 17.7px;
+	color: rgba(0,0,0,0.87);
+	margin-bottom: 8px;
+}
+
+#sponser-card-content-category{
+	font-size: 12px;
+	color: rgba(0,0,0,0.55);
+	margin-bottom: 8px;
+}
+
+#sponser-card-content-waiting{
+	font-size: 12px;
+	color: rgba(0,0,0,0.87);
+	margin-bottom: 4px;
+}
+
+#sponser-card-content-extra{
+	font-size: 12px;
+	color: rgba(0,0,0,0.55);
+}
+
+#sponser-card-subcontent{
```

Comment:
I'm interested to know why you decided to use so many ids instead of using classes for everything :)

Reply:
lol. Don't be interested in my random behavior...   😀 
I just got an error at some point and I changed to ids and everything worked fine... lol..
I'm sure that if I change all back to classes, it will be okay.